src/backend/distributed/commands/utility_hook.c

```diff
@@ -891,7 +891,10 @@ ExecuteDistributedDDLJob(DDLJob *ddlJob)
 				SendCommandToWorkersWithMetadata(setSearchPathCommand);
 			}
 
-			SendCommandToWorkersWithMetadata((char *) ddlJob->commandString);
+			if (ddlJob->commandString != NULL)
+			{
+				SendCommandToWorkersWithMetadata((char *) ddlJob->commandString);
```

Comment:
Optional: I'm wondering whether we should have a separate field for the metadata syncing, e.g. `ddlJob->metadataSyncCommand` that can sometimes be NULL.

Reply:
From what I checked, current `ddlJob->commandString` is already used only as `metadataSyncCommand` in terms of functionality, so we might as well rename `commandString` to `metadataSyncCommand`

Reply:
Would be nice

Reply:
Let me open a separate PR for that rename, seems out of scope for this PR.